Misc/NEWS.d/next/Documentation/2019-01-21-14-30-59.bpo-35803.yae6Lq.rst

```diff
@@ -0,0 +1,2 @@
+Document and test that ``tempfile`` functions may take a ``dir`` which
```

Comment:
Maybe rephrase this to 

```suggestion
Document and test that ``tempfile`` functions may accept a
:term:`path-like object` for the ``dir`` argument.  Patch by Anthony Sottile
```

right now it sounds like the `dir` argument is newly added to the functions.

Reply:
+1 on this comment, LGTM otherwise.